checker/manual/introduction.tex

```diff
@@ -662,6 +662,20 @@ \subsection{Summary of command-line options\label{checker-options}}
   Section~\ref{compiling-libraries}.
 \end{itemize}
 
+Inference
+\begin{itemize}
+\item \<-AinferSignatures>
+  Output suggested annotations for method signatures and fields.
+  These annotations may reduce the number of type-checking
+  errors when running type-checking in the future; see
+  Section~\ref{signature-inference}.
+\item \<-ArelaxedSignatureInference>
```

Comment:
It would be more symmetric if these options were `-AinferSignatures` and `-AinferRelaxedSignatures` or, alternatively, `-AsignatureInference` and `-ArelaxedSignatureInference`. I like the first option better.